src/Resources/Resources.Test/ScenarioTests/PolicyTests.ps1

```diff
@@ -559,6 +559,18 @@ function Test-PolicyAssignmentWithParameters
     Assert-AreEqual $array[0] $expected.Properties.Parameters.listOfAllowedLocations.Value[0]
     Assert-AreEqual $array[1] $expected.Properties.Parameters.listOfAllowedLocations.Value[1]
 
+	# update parameters
+    # this is validation for https://github.com/Azure/azure-powershell/issues/6055
+    $actual = Set-AzPolicyAssignment -Name testPAWP -Scope $rg.ResourceId -PolicyParameter '{ "listOfAllowedLocations": { "value": [ "something", "something else" ] } }'
```

Comment:
@mentat9 it looks like new cmdlet calls were added to this test, but there's no update to the corresponding `.json` recording file -- you will need to re-record this test to ensure that the correct recordings are being used in playback

Reply:
Ah, forgot to copy those over, thanks.